### DISCUSSION TITLE
FastAPI Mkdocs and 1280px dektop screen

### DISCUSSION THREAD

**Discussion by FabriceSalvaire** [fastapi/fastapi#11521] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
...
```

### Description

...

### Operating System

Other

### Operating System Details

...

### FastAPI Version

...

### Pydantic Version

...

### Python Version

...

### Additional Context

With the actual media query CSS we have to scale down the page to get the TOC displayed at left.
Else, the doc behaves like on a mobile.
Such issue is annoying because we have to guess what is going on...

**Comment by jgould22** [comment]
Which doc, swagger or redoc?

Either way swagger comes from swagger-ui https://github.com/swagger-api/swagger-ui

Redoc comes from redoc https://github.com/Redocly/redoc

Additionally you can use whatever CSS you want by setting it https://fastapi.tiangolo.com/reference/openapi/docs/#fastapi.openapi.docs

**Comment by FabriceSalvaire** [comment]
No it is fastapi doc generated by MkDocs. CSS layouts 1280 desktop as a HDPI smartphone.

**Comment by YuriiMotov** [comment]
Seems that everything works fine now.
Let's close this for now to keep the discussions clean and organized.
If you still have this problem, please let us know
